sources/packages/backend/apps/api/Dockerfile

```diff
@@ -3,7 +3,6 @@ FROM registry.access.redhat.com/ubi8/nodejs-16:1-5
 
 LABEL maintainer="BC GOV"
 
-
```

Comment:
Was this an intentional modification?

Reply:
no just syncing with theother dockerfiles